src/go/k8s/cmd/main.go

```diff
@@ -388,7 +389,12 @@ func main() {
 
 			cacheRecorder := helmSourceController.MustMakeCacheMetrics()
 			indexTTL := 15 * time.Minute
-			helmIndexCache := helmSourceController.NewCache(0, indexTTL)
+
+			var helmIndexCache helmSourceController.Cache
+			if helmCacheMaxSize > 0 {
+				helmIndexCache = helmSourceController.NewCache(helmCacheMaxSize, 1*time.Minute)
```

Comment:
Why the change from `indexTTL` (15min) to 1m? Can we make a `const` for the cache TTL regardless of what it's set to?

Reply:
Does that mean you're going to change it back to 15m? I still see the diff as setting it to `1m`.